Gems/Multiplayer/Code/Source/MultiplayerStatSystemComponent.h

```diff
@@ -58,9 +58,11 @@ namespace Multiplayer
 
         struct CumulativeAverage
         {
+            using AverageWindowType = AZ::IO::AverageWindow<double, double, AZ::IO::s_statisticsWindowSize>;
+
             AZStd::string m_name;
-            AZ::IO::AverageWindow<double, double, AZ::IO::s_statisticsWindowSize> m_average;
-            AZ::u64 m_sampleCount = 0;
+            AverageWindowType m_average;
+            double m_lastValue = 0.f;
```

Comment:
Drop the .f for a double.. '0.0' or '0.'. Adding the '.f' means you explicitly want a single precision constant.

Reply:
Done